Gems/Atom/RHI/Code/Include/Atom/RHI.Reflect/Bits.h

```diff
@@ -241,7 +242,7 @@ namespace AZ
         */
         template <typename T> inline T DivideByMultiple(T value, size_t alignment)
```

Comment:
look like places where this is called in RHI has been replaced with AZ::DivideAndRoundUp. would it make sense to deprecate this?